docs/REQUIREMENTS.md

```diff
@@ -664,8 +664,15 @@ INGRESS_NGINX_CONFIG:
 ### NFS Subdir External Provisioner - SAS default storage class
 # Updates to support open source Kubernetes 
 NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas
+
+## Logging and Monitoring
+V4M_STORAGECLASS = local-storage
 ```
 
+You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown above to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).
+
+While other storage classes can be used, the `local-storage` class is **recommended** for the Viya Monitoring and Logging tools.
+
```

Comment:
I myself would still call this out as a NOTE to draw attention to the item. But it's fine as is it right now. I'll approve.